src/store/hotspotDetails/hotspotDetailsSlice.ts

```diff
@@ -1,10 +1,12 @@
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit'
 import { HotspotReward, HotspotRewardSum, Hotspot } from '@helium/http'
+import { WitnessSum } from '@helium/http/build/models/WitnessSum'
```

Comment:
I forgot to export this one in the index of helium-js, I'll make a PR to add that for the future

Reply:
https://github.com/helium/helium-js/pull/141